modules/passkey/test/utils/safe.ts

```diff
@@ -0,0 +1,45 @@
+import { Hex } from '@noble/curves/abstract/utils'
```

Comment:
Can this be imported from the 4337 package?

Reply:
Can you specify the path?

Reply:
I think he means the whole test/utils/safe.ts file.

You should be able to import it from:

```ts
import { buildSafeTransaction, buildSafeTransactionData, DOMAIN_SEPARATOR_TYPEHASH } from '@safe-global/safe-4337/test/utils/safe' 
```

Reply:
Note that that file is specific to user operations, while here we are interacting with the Safe _without_ 4337. I don't think the aforementioned module contains the code that is used here, but that is rather included in the `@safe-global/safe-contracts` package (which unfortunately targets the wrong `ethers` version).

My intuition is that this module is needed, but I didn't dig too deeply.